sunpy_sphinx_theme/conf.py

```diff
@@ -22,7 +22,11 @@ def page_url(page):
     return urljoin(sunpy_website_url_base, page)
 
 
-html_sidebars = {"**": ["docsidebar.html"]}
+html_sidebars = {
+    #  We use a custom sidebar for the gallery.
+    "generated/gallery/index": ["gallery_sidebar.html"],
```

Comment:
this wont pick up examples?

Reply:
You mean if you are on the example?

Reply:
You can have a look at the example docs, they arent as deep as the sunpy ones but should be enough for now.

Reply:
Fixed.